### PR TITLE
Smashing

### PR DESCRIPTION
**Pull request by AndreasGalster**
Dear TSE code reviewer, 

this is the PR for the Smashing Magazine design teardown.

Some notes on implementation:
- We're using css imports to load css files (and split them up)... we know not best practice in a project (multiple downloads) but in a real webpack/sass project you'd get one bundled up css file
- We're using a lot of layout techniques such as flex, floats, grids to design the page
- To run it on your local, run `yarn` and then `yarn start`

### CODE REVIEW

**Comment by umairarshadbutt** [comment]
##
## Pull Request under review by Umair Arshad (TSE)

File: index.html [hunk 1]
```diff
@@ -7,9 +7,485 @@
...
+            <div><i>#</i>Design<span>232</span></div>
+          </li>
+        </ul>
+        <a class="arrow" href="#"></a>
+      </article>
+    </div>
+
+    <div id="latest-articles">
+
+      <div class="latest-article-detail">
```
Comment (umairarshadbutt) [inline index.html]: Use article tag

File: index.html [hunk 2]
```diff
@@ -7,9 +7,485 @@
...
+        <h4>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Ipsum, molestias dolorem amet error incidunt iure!
+        </h4>
+        <h6>Andreas Galster</h6>
+        <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Voluptas repellat reprehenderit quod pariatur
+          ratione
+          rem nemo soluta ipsum repellendus, laudantium debitis deserunt necessitatibus aut id excepturi quas velit quia
+          dicta. Esse id repellendus eaque corporis. Accusamus, culpa. Inventore nisi, aliquid amet fugit sunt animi,
+          accusantium sapiente quas unde impedit recusandae?</p>
+      </div>
+      <div class="latest-article-detail">
```
Comment (umairarshadbutt) [inline index.html]: Use article tag

File: index.html [hunk 3]
```diff
@@ -7,9 +7,485 @@
...
+        <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Voluptas repellat reprehenderit quod pariatur
+          ratione
+          rem
+          nemo soluta ipsum repellendus, laudantium debitis deserunt necessitatibus aut id excepturi quas velit quia
+          dicta.
+          Esse id repellendus eaque corporis. Accusamus, culpa. Inventore nisi, aliquid amet fugit sunt animi,
+          accusantium
+          sapiente quas unde impedit recusandae?</p>
+      </div>
+      <div class="latest-article-detail">
```
Comment (umairarshadbutt) [inline index.html]: latest-article-detail

File: index.html [hunk 4]
```diff
@@ -7,9 +7,485 @@
...
+      <p>Our new book on how to boost business KPIs, build trust and address doubts effectively — without alienating
+        people along
+        the way. By Paul Boag.</p>
+      <a href="#" class=" cta-button">Jump to table of contents →</a>
+      <img src="https://via.placeholder.com/150" alt="">
+    </div>
+
+    <div id="community-content">
+      <div>
+        <div class="item">
```
Comment (umairarshadbutt) [inline index.html]: Use article tag in **community-content** items

File: index.html [hunk 5]
```diff
@@ -7,9 +7,485 @@
...
+          design system to create great digital products. With common traps, gotchas and lessons learned.</p>
+        <a href="#" class=" cta-button">Jump to table of contents ↬</a>
+      </div>
+      <div class="right">
+        <img src="https://via.placeholder.com/150x230" alt="book">
+      </div>
+    </div>
+
+    <div id="guides">
+      <div class="item">
```
Comment (umairarshadbutt) [inline index.html]: Use article tag in **guides** items

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi @AndreasGalster and @rukundoeric
Such a remarkable project you made :+1: ! Awesome README :100: !
We, however, do have some few changes to carry out before we proceed (see review comment).

##

 For this project, we have to make sure nothing is messed up. Kindly compare the screenshots below 👇 and work on the suggested changes.
- [ ] Use article tag for articles
Explanation: You used div tags for articles in your HTML file. For more understanding, you can visit [smashingmagazine.com](https://www.smashingmagazine.com/). You can use the google inspect tool for understanding how the original webpage is using the article tag


### Navbar
- [ ] Mangae **Search Articles** input tag within navbar
- [ ] **[OPTIONAL]** Use the navbar tag (`nav`) for navbar

|[Source Page](https://www.smashingmagazine.com/)|[Clone Page](https://raw.githack.com/AndreasGalster/mv-smashing/smashing/index.html)|
|:---:|:---:|
|source-page|clone|

##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you stick at any point or have any questions, let me know on slack at any time. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Comment by AndreasGalster** [comment]
We've made the changes requests, please check again and let us know what you think :).

**Comment by umairarshadbutt** [comment]
##
## Pull Request under review by Umair Arshad (TSE)

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi @AndreasGalster and @rukundoeric
Such a remarkable project you made :+1: ! Awesome README :100: !
We, however, do have some few changes to carry out before we proceed (see review comment).

##

 For this project, we have to make sure nothing is messed up. Kindly compare the screenshots below 👇 and work on the suggested changes.
### Navbar
- [x] Mangae **Search Articles** input tag within navbar

|[Source Page](https://www.smashingmagazine.com/)|[Clone Page](https://raw.githack.com/AndreasGalster/mv-smashing/smashing/index.html)|
|:---:|:---:|
|source-page|clone|

##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you stick at any point or have any questions, let me know on slack at any time. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Comment by AndreasGalster** [comment]
@umairarshadbutt Hi Umair, we've made the fixes to the navbar. I think the article issue was already addressed in @rukundoeric's [commit](https://github.com/AndreasGalster/mv-smashing/pull/2/commits/5ce712f4613252e955796ccfb592b6a2d02b6e7a)

For reference the fix for the navigation in a GIF:

Op9yq0AbjF

The live link might still be cached as the old version, so it might take a while for the changes to reflect.

**Review by acushlakoncept — APPROVED** [review]
# Status: PROJECT APPROVED  🎉 🚀

### Hi @AndreasGalster and @rukundoeric

You've got an impressive project here, great job. 👏 👏  
Congratulations 🥇 on completing your project and taking the time to implement the changes requested by the TSE.  

I will go ahead and mark this as complete, feel free to merge when you are ready. 

### Cheers and Happy coding! 💻 👏

(Feel free to contact me on slack(@acushla), if you have any additional question)
Uduak Essien
[Twitter](https://twitter.com/acushlakoncept) | [LinkedIn](https://www.linkedin.com/in/acushlakoncept/)

**Pull request merged**